neptune/management/internal/api.py

```diff
@@ -115,9 +110,15 @@ def get_project_list(api_token: Optional[str] = None) -> List[str]:
 
 
 @with_api_exceptions_handler
+def _get_projects(params, api_token: Optional[str] = None) -> List:
```

Comment:
Let's structure it the same way as `_create_project` and `_get_workspace_id` (backend client as an input).